lib/gui/lvgl/Kconfig

```diff
@@ -21,6 +22,14 @@ config APP_LINK_WITH_LVGL
 	  disabled if the include paths for LVGL are causing aliasing
 	  issues for 'app'.
 
+config LVGL_USER_CONFIG_FILE
+    bool "Use user config file"
+    default n
```

Comment:
All boolean configs are `n` by default. This is not needed.

Reply:
ok